inspirehep/modules/authors/rest/publications.py

```diff
@@ -96,14 +97,17 @@ def serialize(self, pid, record, links_factory=None):
             try:
                 publication['journal'] = {}
                 publication['journal']['title'] = result_source.get(
-                    'publication_info', [])[0]['journal_title']
+                    'publication_info', []
```

Comment:
Hmmm, if splitting a function on multiple lines, I think it's a lot more readable if you put each param on it's own line.